app/plugins/adp-backend/src/service/armsLengthBodyRouter.ts

```diff
@@ -41,33 +45,49 @@ export async function createAlbRouter(
   });
 
   router.get('/armsLengthBody', async (_req, res) => {
-    const data = await armsLengthBodiesStore.getAll();
-    res.json(data);
+    try {
+      const data = await armsLengthBodiesStore.getAll();
+      res.json(data);
+    } catch (error) {
+      const errMsg = (error as Error).message;
```

Comment:
I think logging the error object rather than its message property would be better, that way we wont lose the stack trace and debugging would be easier

Reply:
Alternatively, logging the `stack` property will give more detail than just the `message`

Reply:
Like this: 

 const errMsg = (error as Error).message;
      logger.error('Error in updating a arms length body: ', error);
      throw new InputError(errMsg);

Reply:
If that works, yeah. If not, then something like
```
const strongError = (error as Error);
logger.error('Error in updating a arms length body: ', strongError.stack);
throw new InputError(strongError.message);
```

Reply:
I realised that stack is an optional field in Error - in that case would just returning error be better? 

interface Error {
    name: string;
    message: string;
    stack?: string;
}